README.md

```diff
@@ -1,21 +1,39 @@
 # Maslow
 
-Maslow is a tool to create and manage needs. It's a Rails app which is
-part of the GOV.UK Publishing Platform.
+Maslow is a tool to create and manage user needs.
 
-## Dependencies
 
-- Ruby
-- Bundler
-- NodeJS
-- Yarn
-- A running instance of the [Publishing API](https://github.com/alphagov/publishing-api)
+## Technical documentation
 
-## User accounts
+This is a Ruby on Rails app, and should follow [our Rails app conventions][].
 
-Authentication is provided by the [GDS-SSO](https://github.com/alphagov/gds-sso) gem, and in the production environment an instance of [Signon](https://github.com/alphagov/signon) must be running in order to sign in.
+You can use the [GOV.UK Docker environment][] to run the application and its tests with all the necessary dependencies.  Follow [the usage instructions][] to get started.
+
+**Use GOV.UK Docker to run any commands that follow.**
+
+[our Rails app conventions]: https://docs.publishing.service.gov.uk/manual/conventions-for-rails-applications.html
+[GOV.UK Docker environment]: https://github.com/alphagov/govuk-docker
+[the usage instructions]: https://github.com/alphagov/govuk-docker#usage
+
+### Testing
+
+The default `rake` task runs all the tests and records code coverage:
+
+```sh
+bundle exec rake
+```
+
+After running the tests, the `coverage/` folder contains the generated report.
+
+### User accounts
```

Comment:
Minor: we've deleted this in other READMEs, as it's common across our repos and also works out-of-the-box.